index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+
+<html Lang="en-GB">
+
+<head>
+	
+	<link rel="stylesheet" type= "text/css" href="styles.css">
+	
+	<title>Ollie Ritchie, Portfolio</title>
+	
+</head>
+
+<body>
+	
+	<nav>
+		<div class="nav-bar">
+			<a href="#">About</a>
+			<a href="#">Portfolio</a>
+			<a href="#">Contact</a>
+		</div>
+		<button class=cv-download-button>Download CV</button>
+	</nav>
+	
+	<header class="welcome"> <!-- section contaning h1 title and a full width image -->
```

Comment:
your class name could be more semantic eg header-section, but do you really need a class here? Could you not select this section using ' header' instead?

Reply:
Just checked - dont think you're using the class welcome, you are selecting this using header so likely safe to remove the class